.github/workflows/import_smart_workflow.yaml

```diff
@@ -0,0 +1,45 @@
+name: 🔄 Import Smart Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      workflow-id:
+        description: ID of the workflow to import
+        required: true
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PR-Pilot-AI/smart-workflows
+        uses: actions/checkout@v2
+        with:
+          repository: PR-Pilot-AI/smart-workflows
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Determine workflow file path
+        id: filepath
+        run: |
+          if [ -f tools/${{ github.event.inputs.workflow-id }}/workflow.yaml ]; then
+            echo "::set-output name=path::tools/${{ github.event.inputs.workflow-id }}/workflow.yaml"
+          elif [ -f automations/${{ github.event.inputs.workflow-id }}/workflow.yaml ]; then
+            echo "::set-output name=path::automations/${{ github.event.inputs.workflow-id }}/workflow.yaml"
+          else
+            echo "Path not found" && exit 1
+          fi
+      - name: Copy workflow file
+        id: print-workflow
+        run: cat ${{ steps.filepath.outputs.path }}
+      - name: Import Workflow
+        uses: PR-Pilot-AI/smart-actions/quick-task@v1
+        with:
+          api-key: ${{ secrets.PR_PILOT_API_KEY }}
+          agent-instructions: |
+            Create a new file in `.github/workflows/` for the following workflow:
+
+            ---
+            
+            ${{ steps.print-workflow.outputs.path }}
```

Comment:
/pilot I don't think this is the right variable. I want this to be the output of `cat` in the previous step